<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>bioinformatics</title>
    <link href="https://fonts.googleapis.com/css?family=Sniglet|Open+Sans:400,700&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="../assets/css/style.css">
</head>
<body>
    <div class="container"><h1><span class = "subreddit">r/bioinformatics </span></h1>
<div class="post">
<h2>Use of Large Language Models in Bioinformatics</h2>
<span class = "user">posted by: u/lok_joshi </span><span class = "numcomments">26 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/14ayoss/use_of_large_language_models_in_bioinformatics/" target="_blank">(source)</a>
<button onclick="toggles(12)">View full post</button><div id="12" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>Is anyone aware of the use of LLM in molecular biology? I read somewhere that people use LLMs to predict future COVID variants. What other possible uses of LLMs do you see in life science?  </p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>Agile software development for bioinformatics: have you used it and/or what do you think?</h2>
<span class = "user">posted by: u/astrologicrat </span><span class = "numcomments">11 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/148rglx/agile_software_development_for_bioinformatics/" target="_blank">(source)</a>
<button onclick="toggles(16)">View full post</button><div id="16" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>My jobs have primarily been in academic settings, but this year someone decided to try imposing agile methods/scrum for one of our bioinformatics software development projects. My ~5-person team now has Jira issues, standups, a scrum master, two week sprints, etc...</p>

<p>A few months in and I am absolutely not a fan. We spend half a week planning the sprint, and then day 1 of the sprint I am immediately asked to tackle an issue that wasn&#39;t planned, forcing me to choose between getting &quot;credit&quot; for an issue on the Jira board or the ad hoc request. The daily standups feel like micromanagement, while the scrum master feels like someone trying to be a second boss. I am pinged constantly to close tickets for a burndown chart that really hasn&#39;t helped anything, and deadlines are being set for individual tasks down to the day, giving me no flexibility when it comes to working on tasks or allocating my own time.</p>

<p>My background is mostly aligned with R&amp;D where strict estimations usually don&#39;t hold up. Also, while I am quite happy to work on teams, I am usually afforded quite a bit of independence as a Ph.D.-level scientist; now I feel like I&#39;ve been reduced to an assembly line worker.</p>

<p>Aside from venting, I do want to hear 1) if you have had experience with agile anything in a bioinformatics context and 2) what your opinion of it is.</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>Bioinformatics to Data Science</h2>
<span class = "user">posted by: u/kbabqiqja </span><span class = "numcomments">3 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/13dfwcj/bioinformatics_to_data_science/" target="_blank">(source)</a>
<button onclick="toggles(63)">View full post</button><div id="63" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>[removed]</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>How to get fast omics training?</h2>
<span class = "user">posted by: u/amebied </span><span class = "numcomments">15 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/1377b09/how_to_get_fast_omics_training/" target="_blank">(source)</a>
<button onclick="toggles(70)">View full post</button><div id="70" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>Hello,</p>

<p>I am a physician and a PhD student at a US-based college of Public health. I am interested in studying somatic mutations and their effects on cancer outcomes. I have heard of the TOPMed database and would like to apply for a TOPMed Fellowship. The problem is that I am just starting to build the bioinformatics knowledge and skills to research using that database. I have a deadline of July 31st to submit a proposal, and I need to show that I can do research using that database. Do you have any suggestions on resources that I can use (even as a self-learner) to accelerate my path to omics training? Thanks.</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>I wrote a blog post about how to conduct omics data analysis in R with ChatGPT prompts. I created a volcano plot using the GEO microarray dataset as an example this time.</h2>
<span class = "user">posted by: u/BioAdvantage </span><span class = "numcomments">14 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/12z7t9j/i_wrote_a_blog_post_about_how_to_conduct_omics/" target="_blank">(source)</a>
<button onclick="toggles(86)">View full post</button><div id="86" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>[removed]</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>PLOS Computational Biology: "Ten Simple Rules" papers</h2>
<span class = "user">posted by: u/tb877 </span><span class = "numcomments">9 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/12puflj/plos_computational_biology_ten_simple_rules_papers/" target="_blank">(source)</a>
<button onclick="toggles(95)">View full post</button><div id="95" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>Just found out about <a href="https://collections.plos.org/collection/ten-simple-rules/">this series of papers</a> from PLOS about a variety of subject they don’t necessarily teach you in grad school. Thought I’d share this here, definitely looks interesting.</p>

<p>Here’s a list of the ones I’ve personally added to my Zotero library, but you can find even more through the link above.</p>

<p>---</p>

<ol>
<li><p>Bourne, P. E. &amp; Korngreen, A. Ten Simple Rules for Reviewers. PLOS Computational Biology 2, e110 (2006).</p></li>
<li><p>Lonsdale, A., Penington, J. S., Rice, T., Walker, M. &amp; Dashnow, H. Ten Simple Rules for a Bioinformatics Journal Club. PLOS Computational Biology 12, e1004526 (2016).</p></li>
<li><p>Gaëta, B. A. et al. Ten simple rules for forming a scientific professional society. PLOS Computational Biology 13, e1005226 (2017).</p></li>
<li><p>Bruckmann, C. &amp; Sebestyén, E. Ten simple rules to initiate and run a postdoctoral association. PLOS Computational Biology 13, e1005664 (2017).</p></li>
<li><p>Erren, T. C., Slanger, T. E., Groß, J. V., Bourne, P. E. &amp; Cullen, P. Ten Simple Rules for Lifelong Learning, According to Hamming. PLOS Computational Biology 11, e1004020 (2015).</p></li>
<li><p>Méndez, M. Ten simple rules for developing good reading habits during graduate school and beyond. PLOS Computational Biology 14, e1006467 (2018).</p></li>
<li><p>Bourne, P. E. Ten Simple Rules for Getting Published. PLOS Computational Biology 1, e57 (2005).</p></li>
<li><p>Bourne, P. E. Ten Simple Rules for Making Good Oral Presentations. PLOS Computational Biology 3, e77 (2007).</p></li>
<li><p>Erren, T. C. &amp; Bourne, P. E. Ten Simple Rules for a Good Poster Presentation. PLOS Computational Biology 3, e102 (2007).</p></li>
<li><p>Pautasso, M. Ten Simple Rules for Writing a Literature Review. PLOS Computational Biology 9, e1003149 (2013).</p></li>
<li><p>Zhang, W. Ten Simple Rules for Writing Research Papers. PLOS Computational Biology 10, e1003453 (2014).</p></li>
<li><p>Ekins, S. &amp; Perlstein, E. O. Ten Simple Rules of Live Tweeting at Scientific Conferences. PLOS Computational Biology 10, e1003789 (2014).</p></li>
<li><p>Rougier, N. P., Droettboom, M. &amp; Bourne, P. E. Ten Simple Rules for Better Figures. PLOS Computational Biology 10, e1003833 (2014).</p></li>
<li><p>Weinberger, C. J., Evans, J. A. &amp; Allesina, S. Ten Simple (Empirical) Rules for Writing Science. PLOS Computational Biology 11, e1004205 (2015).</p></li>
<li><p>Bourne, P. E., Polka, J. K., Vale, R. D. &amp; Kiley, R. Ten simple rules to consider regarding preprint submission. PLOS Computational Biology 13, e1005473 (2017).</p></li>
<li><p>Mensh, B. &amp; Kording, K. Ten simple rules for structuring papers. PLOS Computational Biology 13, e1005619 (2017).</p></li>
<li><p>Noble, W. S. Ten simple rules for writing a response to reviewers. PLOS Computational Biology 13, e1005730 (2017).</p></li>
<li><p>Peterson, T. C., Kleppner, S. R. &amp; Botham, C. M. Ten simple rules for scientists: Improving your writing productivity. PLOS Computational Biology 14, e1006379 (2018).</p></li>
<li><p>Marai, G. E., Pinaud, B., Bühler, K., Lex, A. &amp; Morris, J. H. Ten simple rules to create biological network figures for communication. PLOS Computational Biology 15, e1007244 (2019).</p></li>
<li><p>Cheplygina, V., Hermans, F., Albers, C., Bielczyk, N. &amp; Smeets, I. Ten simple rules for getting started on Twitter as a scientist. PLOS Computational Biology 16, e1007513 (2020).</p></li>
<li><p>Prlić, A. &amp; Procter, J. B. Ten Simple Rules for the Open Development of Scientific Software. PLOS Computational Biology 8, e1002802 (2012).</p></li>
<li><p>Perez-Riverol, Y. et al. Ten Simple Rules for Taking Advantage of Git and GitHub. PLOS Computational Biology 12, e1004947 (2016).</p></li>
<li><p>List, M., Ebert, P. &amp; Albrecht, F. Ten Simple Rules for Developing Usable Software in Computational Biology. PLOS Computational Biology 13, e1005265 (2017).</p></li>
<li><p>Taschuk, M. &amp; Wilson, G. Ten simple rules for making research software more robust. PLOS Computational Biology 13, e1005412 (2017).</p></li>
<li><p>Lee, B. D. Ten simple rules for documenting scientific software. PLOS Computational Biology 14, e1006561 (2018).</p></li>
<li><p>Corpas, M., Gehlenborg, N., Janga, S. C. &amp; Bourne, P. E. Ten Simple Rules for Organizing a Scientific Meeting. PLOS Computational Biology 4, e1000080 (2008).</p></li>
<li><p>Bateman, A. &amp; Bourne, P. E. Ten Simple Rules for Chairing a Scientific Session. PLOS Computational Biology 5, e1000517 (2009).</p></li>
<li><p>Gu, J. &amp; Bourne, P. E. Ten Simple Rules for Graduate Students. PLOS Computational Biology 3, e229 (2007).</p></li>
<li><p>Marino, J., Stefan, M. I. &amp; Blackford, S. Ten Simple Rules for Finishing Your PhD. PLOS Computational Biology 10, e1003954 (2014).</p></li>
<li><p>Vicens, Q. &amp; Bourne, P. E. Ten Simple Rules for a Successful Collaboration. PLOS Computational Biology 3, e44 (2007).</p></li>
<li><p>Bourne, P. E. &amp; Chalupa, L. M. Ten Simple Rules for Getting Grants. PLOS Computational Biology 2, e12 (2006).</p></li>
<li><p>Bourne, P. E. &amp; Friedberg, I. Ten Simple Rules for Selecting a Postdoctoral Position. PLOS Computational Biology 2, e121 (2006).</p></li>
<li><p>Bourne, P. E. &amp; Barbour, V. Ten Simple Rules for Building and Maintaining a Scientific Reputation. PLOS Computational Biology 7, e1002108 (2011).</p></li>
<li><p>Tomaska, L. &amp; Nosek, J. Ten simple rules for writing a cover letter to accompany a job application for an academic position. PLOS Computational Biology 14, e1006132 (2018).</p></li>
<li><p>Sura, S. A. et al. Ten simple rules for giving an effective academic job talk. PLOS Computational Biology 15, e1007163 (2019).</p></li>
<li><p>Tregoning, J. S. &amp; McDermott, J. E. Ten Simple Rules to becoming a principal investigator. PLOS Computational Biology 16, e1007448 (2020).</p></li>
<li><p>Yuan, K., Cai, L., Ngok, S. P., Ma, L. &amp; Botham, C. M. Ten Simple Rules for Writing a Postdoctoral Fellowship. PLOS Computational Biology 12, e1004934 (2016).</p></li>
<li><p>Bourne, P. E. &amp; Chalupa, L. M. Ten Simple Rules for Getting Grants. PLOS Computational Biology 2, e12 (2006).</p></li>
<li><p>Mensh, B. &amp; Kording, K. Ten simple rules for structuring papers. PLOS Computational Biology 13, e1005619 (2017).</p></li>
<li><p>Weinberger, C. J., Evans, J. A. &amp; Allesina, S. Ten Simple (Empirical) Rules for Writing Science. PLOS Computational Biology 11, e1004205 (2015).</p></li>
</ol>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>Many bioinformatics programming tasks can be automated with ChatGPT</h2>
<span class = "user">posted by: u/Budget_Peak_1778 </span><span class = "numcomments">53 comments </span><a href="https://arxiv.org/abs/2303.13528" target="_blank">(source)</a>
<button onclick="toggles(103)">View full post</button><div id="103" class="mdwrapper"><p>NO BODY FOR THE SUBMISSION</p></div>
</div>
<div class="post">
<h2>R packages on Linux... It is a struggle</h2>
<span class = "user">posted by: u/Z3ratoss </span><span class = "numcomments">44 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/129ft2o/r_packages_on_linux_it_is_a_struggle/" target="_blank">(source)</a>
<button onclick="toggles(140)">View full post</button><div id="140" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>I just spend multiple hours getting <a href="https://www.archrproject.com/">ArchR</a> installed on my CentOS system.</p>

<p>It is always such a pain to install big R projects.</p>

<p>If they are on conda / bioconductor I can grab them with mamba.
But when the compiling from github/cran starts and I know I will have to fix dependencies one by one and read through DESCRIPTION files :(</p>

<p>I tried to use the binary cran mirror <a href="https://packagemanager.rstudio.com/">https://packagemanager.rstudio.com/</a> but this does not install system dependencies correctly.</p>

<p>There is also this <a href="https://github.com/Enchufa2/rspm">https://github.com/Enchufa2/rspm</a> but it does not play nicely with conda .</p>

<p>Any suggestions?</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>What are the struggles that bioinformatician face or overwhelm them?</h2>
<span class = "user">posted by: u/Alert_Faithlessness </span><span class = "numcomments">39 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/12402xq/what_are_the_struggles_that_bioinformatician_face/" target="_blank">(source)</a>
<button onclick="toggles(156)">View full post</button><div id="156" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>I am new to this field, writing a grant. I am a biologist, but the project I ended up getting a grant on will need a lot of bioinformatics work. I am trying to see what are the struggles bioinformatician face or what overwhelm them. Your response will help me with interviewing candidates.</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>Structure based drug design and machine learning vs. deep learning models</h2>
<span class = "user">posted by: u/Astatinee </span><span class = "numcomments">6 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/11jdyt3/structure_based_drug_design_and_machine_learning/" target="_blank">(source)</a>
<button onclick="toggles(184)">View full post</button><div id="184" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>Hello there fellow bioinformaticians,</p>

<p>I have recently generated ML models (scikit-learn) for a drug design project and got reasonable r-squared values of up to 0.67. Now, I was wondering if someone has experience with ML for drug design and has attempted DL for model improvements. I would like to improve my predictions but feel like I have reached the limit of ML.</p>

<p>Some background: The enzyme target is Matrix Metalloproteinase 9, which is involved in extracellular matrix remodelling pathways. Overexpression has been linked to physiological diseases, i.e. cancer, fibrosis. There are a few drugs in clinical trials but these most likely also interact with other matrix metalloproteinases which makes this a pretty difficult active site to target. There are some other difficulties for the effective drug design against MMP9 but I won&#39;t go into these. Anyway, this issue with drug specificity warrants more structure based drug design to improve target specificity. So, since this is an interesting target from a biological basis and the ChEMBL database (<a href="https://www.ebi.ac.uk/chembl/">https://www.ebi.ac.uk/chembl/</a>) has a dataset on bioactive molecules, I thought I would attempt to build some ML models.</p>

<p>The original models were built using PaDEL descriptors which yielded r-squared values of around 0.55. I wanted to improve this and supplemented the PaDEL feature list with AutoDock Vina affinity parameters and some Lipinski properties. These models got r-squared values of up to 0.67. I was honestly pretty surprised that these would improve the model scores like they did but I am now looking to push these a bit more. So here are my questions, has anybody approached initial drug design like this and ended up using deep learning models? And, what kind of model improvements could I expect? Is it worth it to learn deep learning libraries (TensorFlow) to improve on ML scores?</p>

<p>PS.: I do this in my free time so, feel free to dm me. I am happy to share the code and answer any questions. Also, I&#39;m very open to suggestions.</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>The struggle of an open source maintainer – not bioinformatics but heartfelt</h2>
<span class = "user">posted by: u/bioinformat </span><span class = "numcomments">5 comments </span><a href="https://github.com/zloirock/core-js/blob/master/docs/2023-02-14-so-whats-next.md" target="_blank">(source)</a>
<button onclick="toggles(239)">View full post</button><div id="239" class="mdwrapper"><p>NO BODY FOR THE SUBMISSION</p></div>
</div>
<div class="post">
<h2>From bioinformatics to tool dev</h2>
<span class = "user">posted by: u/Bad-Tuchus </span><span class = "numcomments">16 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/10rx795/from_bioinformatics_to_tool_dev/" target="_blank">(source)</a>
<button onclick="toggles(250)">View full post</button><div id="250" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>I am a PhD grad student in bioinformatics with experience in development and maintenance of databases and genome browsers. I also have experience in pipeline dev, genome assembly, annotation and other downstream steps. I’ve also used machine learning for genomic feature extraction. 
I really enjoy(ed) working on the various projects I am/was involved in, but I like building products (databases, web apps and such) the most. What I really want to work on in the future is bioinformatics tool development, but that might be a bit of a hurdle for me since I’m not from a computer science background. 
I would really appreciate any advice on how to make this transition.</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>Bionformatics tools for the future</h2>
<span class = "user">posted by: u/LifeStructure </span><span class = "numcomments">9 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/10rbr8a/bionformatics_tools_for_the_future/" target="_blank">(source)</a>
<button onclick="toggles(255)">View full post</button><div id="255" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>Hello,</p>

<p>I just started my PhD in Biology in a lab that is very computationally oriented (although we do some wet work). I am currently analyzing peptide and DNA sequences using Python. </p>

<p>&#x200B;</p>

<p>My question is: what kind of skills (both soft and hard) would me make a competitive candidate for work (in industry) when I am done with my PhD. My main focus is my research and everything related, but I would like to work on other skills in my down time.</p>

<p>&#x200B;</p>

<p>Any insight would be appreciated.</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>A universal deep-learning model for zinc finger design enables transcription factor reprogramming</h2>
<span class = "user">posted by: u/Robert_Larsson </span><span class = "numcomments">7 comments </span><a href="https://www.nature.com/articles/s41587-022-01624-4" target="_blank">(source)</a>
<button onclick="toggles(261)">View full post</button><div id="261" class="mdwrapper"><p>NO BODY FOR THE SUBMISSION</p></div>
</div>
<div class="post">
<h2>A concise list of skills/competencies required by the computational biologist or bioinformatician</h2>
<span class = "user">posted by: u/None </span><span class = "numcomments">37 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/10ld3qc/a_concise_list_of_skillscompetencies_required_by/" target="_blank">(source)</a>
<button onclick="toggles(266)">View full post</button><div id="266" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>a. unix/linux</p>

<p>b. programming language such as Python</p>

<p>c. statistical language such as R</p>

<p>d. molecular biology / genetics / biochemistry / epigenetics</p>

<p>e. statistics</p>

<p>f. higher order mathematics such as linear algebra</p>

<p>g. data structures and algorithms</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>What are some ways that bioinformatics can contribute to the understanding of rare diseases using already available data?</h2>
<span class = "user">posted by: u/MrNezzer </span><span class = "numcomments">7 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/10f6fvx/what_are_some_ways_that_bioinformatics_can/" target="_blank">(source)</a>
<button onclick="toggles(280)">View full post</button><div id="280" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>Hi Everyone --</p>

<p>I&#39;m new to this sub and was just wondering about how bioinformatics techniques can get applied to better understand rare diseases with data that is already available. If you have experience in this particular area of research, your feedback would be very appreciated, but any ideas/opinions are welcome!</p>

<p>Thanks!</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>Leaving academia</h2>
<span class = "user">posted by: u/ClownMorty </span><span class = "numcomments">19 comments </span><a href="/r/bioinformatics/comments/104c24x/leaving_academia/j361yf6/" target="_blank">(source)</a>
<button onclick="toggles(306)">View full post</button><div id="306" class="mdwrapper"><div class="md"><p>It does if you do the keyword suggestion. I am contacted semi regularly for relevant jobs. What I did was find the jobs I wanted and copy pasted dozens of entire job listings into a word cloud generator. I then splashed 10-15 or so of the biggest words all throughout my resume and LinkedIn page. It works the algorithm like magic. By the way, this is one of the tricks those resume review companies do to help you, so why not cut out the the middle man!</p>
</div></div>
</div>
<div class="post">
<h2>How to use Conda environments? Best practices?</h2>
<span class = "user">posted by: u/No_Touch686 </span><span class = "numcomments">17 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/ywrz6f/how_to_use_conda_environments_best_practices/" target="_blank">(source)</a>
<button onclick="toggles(364)">View full post</button><div id="364" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>I’ve stopped being lazy and jamming everything into my base Conda environment after stuff kept breaking.</p>

<p>But I’m not totally sure what the best practice is for having different environments. I’m aware that it’s good to have an environment for each project, but I’m not clear on other things.</p>

<p>For example, I use R through Conda - should you have R in its own environment? Or should you add to each separate environment?</p>

<p>Ideally I want to keep stuff as simple as possible! Thanks.</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>bioinformatics in psychedelics or psychosomatic medical space</h2>
<span class = "user">posted by: u/shitheadrabbit </span><span class = "numcomments">7 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/yj5jn0/bioinformatics_in_psychedelics_or_psychosomatic/" target="_blank">(source)</a>
<button onclick="toggles(393)">View full post</button><div id="393" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>I looked through the FAQs and believe, though this is a career question, that information on course selection, career prospects, and beginner tools have not adequately answered my question. </p>

<p>I am applying to MD/Ph.Ds and want to invest fully in bioinformatics. I have published work in the clinical space and strong background in clinical trials. I was wondering if anyone in the fold is aware of institutions with strong backgrounds in psychiatric bioinformatics, particularly in psychedelic research. I previously worked at JHU and that space was extremely underdeveloped.  Other than Diesseroth I&#39;m racking my brains over finding similar labs. Anyone have any pointers on how to look?</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>Are data engineering skills useful for bioinformaticians (at a Ph.D. student level)?</h2>
<span class = "user">posted by: u/merdanosman </span><span class = "numcomments">8 comments </span><a href="/r/bioinformatics/comments/xwkui4/are_data_engineering_skills_useful_for/ir980qf/" target="_blank">(source)</a>
<button onclick="toggles(416)">View full post</button><div id="416" class="mdwrapper"><div class="md"><p>Definitely yes. Recently i have read an inretesting paper. This git-hub repo is in my mind a good example of what you can achieve with your data science/ML skills. Best luck. </p>

<p><a href="https://github.com/YunxiaoRen/ML-iAMR">https://github.com/YunxiaoRen/ML-iAMR</a></p>
</div></div>
</div>
<div class="post">
<h2>What are the biggest advances in Python for bioinformatics in the last 5 years?</h2>
<span class = "user">posted by: u/Wooden-Sock8699 </span><span class = "numcomments">10 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/xuj039/what_are_the_biggest_advances_in_python_for/" target="_blank">(source)</a>
<button onclick="toggles(419)">View full post</button><div id="419" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>I&#39;m currently reading Vince Buffalo&#39;s <em>Bioinformatics Data Skills</em> (2015) and Buffalo spends nearly 130 pages discussing the relevance of R (especially Bioconductor) packages for bioinformatics, and about 10 on Python (all for quickly going over some Pysam features). But I know some important progress has been made in Python since Buffalo&#39;s book, like the release of the very fast cyvcf2 package for processing VCF files in 2017, or the continual improvement of the Biopython package. What do you think are some of the ways Python has made real advances in, for bioinformatics, in the last 5 years? Is there anything Buffalo would have had to give more consideration to in explaining how to use in Python if he rewrote his book today?</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>Switch from computer science to computational biology</h2>
<span class = "user">posted by: u/DeonTech </span><span class = "numcomments">6 comments </span><a href="/r/bioinformatics/comments/xsh57j/switch_from_computer_science_to_computational/iqkjdkk/" target="_blank">(source)</a>
<button onclick="toggles(420)">View full post</button><div id="420" class="mdwrapper"><div class="md"><p>Yes you absolutely can make the switch!!!! You have the exact same educational background as me (MS in CS with emphasis on ML, Big Data)! I ended up working in industry for 3 years before returning to PhD for Neuroscience. I plan on using my computational skills to accelerate my research interests (memory + disorders)!</p>

<p>That being said, however, I also had research experience in a Bioinformatics lab and internship with a Biopharma company. </p>

<p>So, as you lack any kind of experience in a lab, definitely get some. Not saying that you have to, but that will improve your resume 10x. Your computational skills alone makes you stand out tremendously already (trust me, I get offers every which way because of my experience with big data). I was accepted to a number of Bioinformatics / Comp Bio PhD programs in the USA but I didn&#39;t want to focus on computation as my research. I just wanted to use my skills on top of all the massive datasets that exist out there! </p>

<p>Your options are certainly there! You just have to prove it to the admissions people that you have the chops! So when you start your applications, make sure you talk about the biology sufficiently and how your pure computational background will be a solid foundation for this field.</p>

<p>I feel like applying to MS programs will be a lot easier to get in than PhD, but that is not to deter you. Just to let you know what effort to put forth as you approach application season.</p>
</div></div>
</div>
<div class="post">
<h2>Getting started with Single Cell Genomics analyses</h2>
<span class = "user">posted by: u/roshni-b </span><span class = "numcomments">9 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/v6uwbs/getting_started_with_single_cell_genomics_analyses/" target="_blank">(source)</a>
<button onclick="toggles(512)">View full post</button><div id="512" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>I&#39;m looking for resources (lectures, tool recommendations, tutorials) to learn the following topics, in order to prepare working in a Single-Cell Genomics research laboratory:</p>

<ul>
<li>Gene Set Enrichment Analysis &amp; Gene Ontology Enrichment Analysis</li>
<li>Differential Gene Expression Analysis</li>
<li>Gene Abundance Estimations</li>
<li>Survival Analysis</li>
<li>Calling, Annotating mutations &amp; Genetic aberrations</li>
<li>Statistical Analysis (For eg. Pearson/Spearman correlation, student t-test, wilcox rank sum test, ANOVA, Kruskal-Wallis test)</li>
</ul>

<p>What seminal papers in Single Cell Genomics should I read/reproduce where these concepts have been applied?</p>

<p>Thanks in advance :)</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>Non-fiction book recommendations related to bioinformatics or science (not text books)</h2>
<span class = "user">posted by: u/VforValmont </span><span class = "numcomments">7 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/v0m47u/nonfiction_book_recommendations_related_to/" target="_blank">(source)</a>
<button onclick="toggles(531)">View full post</button><div id="531" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>I am searching for my next book to read and would like something generally related to our field (or science more broadly). What are you recommendations?</p>

<p>Here are some of the books I have recently read and would recommend to anyone one this sub:</p>

<p>The Science of Science - Wang &amp; Barabasi </p>

<p>Covers a pretty new cool field, I also love network science so I really ate this one up. It is definitely not bioinformatics specific, but is very applicable to all of science and academia.</p>

<p>The Genome War - Shreeve</p>

<p>Story of the human genome project, mostly from the Craig Venter side of things. I like to think of it was kind of the origin story of modern bioinformatics and molecular biology. It&#39;s a page turner, I&#39;ve actually read/listened to it twice I liked it so much. </p>

<p>The Structure of Scientific Revolutions  - Kuhn</p>

<p>This is DENSE, but an amazingly influential work! Fun fact, the modern use of the word paradigm is thanks to this book and the phase &quot;paradigm shift&quot; come from this. Since reading this I now see Kuhn&#39;s ideas popping up all over the place.</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>Intersectionality between healthcare providers and bioinformatics?</h2>
<span class = "user">posted by: u/DiscursiveMind </span><span class = "numcomments">12 comments </span><a href="/r/bioinformatics/comments/nrn6iv/intersectionality_between_healthcare_providers/h0j4dgh/" target="_blank">(source)</a>
<button onclick="toggles(639)">View full post</button><div id="639" class="mdwrapper"><div class="md"><p>There are several domains under the <a href="https://translational-medicine.biomedcentral.com/articles/10.1186/1479-5876-8-22/figures/1">biomedical informatics</a> umbrella that overlap with your question.</p>

<p>Bioinformatics is the application of informatics tools and techniques on molecular data.</p>

<p><a href="https://www.amia.org/applications-informatics/translational-bioinformatics">Translational bioinformatics</a> is working on how to take discoveries found on the bench and develop clinical applications at the bed side. There is a lot of genomics and pharmocogenomics occurring in this space right now.</p>

<p><a href="https://www.amia.org/applications-informatics/clinical-informatics">Clinical or Health Informatics</a> is focused on using informatics tools on individual patients and the delivery of healthcare services.</p>

<p><a href="https://www.amia.org/applications-informatics/clinical-research-informatics">Clinical research informatics</a> is largely focused on clinical trials and helping to manage new knowledge about disease and health.</p>

<p><a href="https://www.amia.org/applications-informatics/consumer-health-informatics">Consumer health informatics</a> is looking to improve clinical outcomes through informatics from the patient perspective. This is the area I&#39;m currently working in, as we are building remote patient monitoring for rural and underserved patients.</p>

<p>When you pull back away from individual patients, and start dealing with populations, now you are in the domain of <a href="https://www.amia.org/applications-informatics/public-health-informatics">public health informatics</a>.</p>

<p>As a clinician, you can <a href="https://www.theabpm.org/become-certified/subspecialties/clinical-informatics/">pursue board certification in informatics</a> and there is also a speciality of <a href="https://www.amia.org/programs/working-groups/nursing-informatics">nursing informatics</a> and <a href="https://www.amia.org/programs/working-groups/dental-informatics">dental informatics</a>.</p>

<p>My research interest is established in precision medicine and genomic data interpretation by medical professionals. Even though I&#39;m working on more of a digital health and telemedicine project right now, I hope to fold in some genomics down the road once we get the project off the ground and established at a few different sites.</p>
</div></div>
</div>
<div class="post">
<h2>5 Cool AI-Powered Drug Discovery Tools</h2>
<span class = "user">posted by: u/pasticciociccio </span><span class = "numcomments">0 comments </span><a href="https://medium.com/geekculture/5-cool-ai-powered-drug-discovery-tools-1d7e976ffc2a" target="_blank">(source)</a>
<button onclick="toggles(640)">View full post</button><div id="640" class="mdwrapper"><p>NO BODY FOR THE SUBMISSION</p></div>
</div>
<div class="post">
<h2>Virtual bioinformatics groups?</h2>
<span class = "user">posted by: u/mathymate </span><span class = "numcomments">20 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/ntuy5o/virtual_bioinformatics_groups/" target="_blank">(source)</a>
<button onclick="toggles(641)">View full post</button><div id="641" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>Hi, are there any open or invite-only bioinformatics groups on Discord or Slack where people critique each other projects, provide resources, and help with coding issues? I keep getting stuck at some points in coding in R with applications to computational biology and bioinformatics, and some kind of help group would be nice. Thanks!</p>

<p>Edit:</p>

<p>Also, a resource to finding datasets would be helpful as well. A lot of the guides I found use datasets within the libraries and don&#39;t explain how to find user-friendly datasets. </p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>Beginner Bioinformatics Project and some Tips to pursue a career</h2>
<span class = "user">posted by: u/RepresentativeYear72 </span><span class = "numcomments">19 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/nqouh7/beginner_bioinformatics_project_and_some_tips_to/" target="_blank">(source)</a>
<button onclick="toggles(642)">View full post</button><div id="642" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>Hey Guys,</p>

<p>I am a sophomore interested in Bio-Informatics. I am currently pursuing Biology Majors and had a few elective courses in Basic Programming with python and a Minor in Physics. I am quite new to this field. I am comfortable with Python and use Arch, so I know bash as well. I am learning by watching some courses in Bioinformatics on youtube. I want to do a Project side by side as I am learning, so can you suggest some papers or projects that I can do to implement what I am learning by recreating the results. Also, I would like to know if you have some tips that might help me in the future career.</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>Motivational post for newbies</h2>
<span class = "user">posted by: u/GeneticVariant </span><span class = "numcomments">16 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/mqnpzf/motivational_post_for_newbies/" target="_blank">(source)</a>
<button onclick="toggles(657)">View full post</button><div id="657" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>Sorry if posts like this arent allowed but...</p>

<p>I&#39;ve noticed a common theme of people new to the field  feeling overwhelmed by the decentralised nature of bioinformatics (myself included). I just want to say that it&#39;s totally normal to feel confused by all the jargon and feel incompetent when you just cant get something to work or cant understand a complex concept.</p>

<p>I wanted to make this post to make it clear to people in those situations that you are not alone. Just keep studying those definitions, keep trying different things on your code and follow through those google search rabbit holes. As long as you&#39;re trying, you&#39;re making progress. </p>

<p>Good luck!!</p>

<p>Edit: Thank you for the upvotes and awards!</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>What's your bioinformatics bible? Any bibliography recommendations for experimental design?</h2>
<span class = "user">posted by: u/josealv23 </span><span class = "numcomments">11 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/mt8x69/whats_your_bioinformatics_bible_any_bibliography/" target="_blank">(source)</a>
<button onclick="toggles(662)">View full post</button><div id="662" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>A little bit of background here. I&#39;m a biomedical sciences PhD student, with some basic knowledge on bioinformatics. I landed some time ago in a physicists lab, where they are now testing and trying to transfer their investigation to the clinic, being me the &quot;biothings&quot; person in the middle for preclinical experimentation.</p>

<p>Everything has been going fine so far. However, we came across the conclusion that we could benefit from RNAseq data of the human cells we are using right now, but I&#39;m finding it difficult to conclude drafting the experimental design with things such as numbers of reads, etc. I know they are basic concepts, but I would like to be sure that I&#39;m not missing anything, so that&#39;s why I am looking for some recommended bibliography that could help me verify that everything is okay.</p>

<p>So, what would be your bibliographic recommendations for this matter? And for data analysis? (I&#39;m an R based user, but wouldn&#39;t mind changing).</p>

<p>PS: With bibliography I don&#39;t refer to concrete scientific papers, but to broader academical sources, i.e., textbooks, manuals, etc.</p>

<p>Thanks a lot!!! :)</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>Getting started with NGS data analysis</h2>
<span class = "user">posted by: u/AlfredoMarchetti </span><span class = "numcomments">16 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/lngsqo/getting_started_with_ngs_data_analysis/" target="_blank">(source)</a>
<button onclick="toggles(679)">View full post</button><div id="679" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>Hello everyone, 
I am a medical undergrad and I have recently started appreciating computing solutions in the fields of biology and biomedical science (I’m also very new to Reddit, lol). 
I have some experience with R programming (basic) especially when it comes to microarray DEA. I have no experience with python or other programming languages but I’m willing to learn. </p>

<p>I am now working on a thesis project in the field of leukemia genomics but I have little experience with NGS data outputs and analysis. Since this project is going to last for 2 years or so I’ve decided that I want to acquire new computing skills that may come in handy also in the future. 
However, I am currently struggling finding good materials and guidance in this field. I don’t know about any good online course and I can’t find resources for what I need (an important part of my job would be the graphical elaboration and display of the output data). Can anyone refer me to good and simple resources to work on sequencing data? Any other advice would be great</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>Getting started with NGS data analysis</h2>
<span class = "user">posted by: u/self_root </span><span class = "numcomments">16 comments </span><a href="/r/bioinformatics/comments/lngsqo/getting_started_with_ngs_data_analysis/go0b4rc/" target="_blank">(source)</a>
<button onclick="toggles(680)">View full post</button><div id="680" class="mdwrapper"><div class="md"><p>While it&#39;s difficult to point you towards a resource that gives an overview of this diverse field, I would recommend the book &quot;bioinformatics data skills&quot; by Vince Buffalo to give you a sense of the skillset you&#39;ll need to develop. It doesn&#39;t go much into the actual analysis of genomic data but instead deals with how to manage the data. There are chapters on linux, working on remote machines, git, data types, R, and other very useful topics for working with large scale genomics data. It&#39;d be best if you had some of your own data to get you started, but there are tons of public databases out there (e.g. Gene expression omnibus, or GEO).</p>
</div></div>
</div>
<div class="post">
<h2>learning resources on GWAS</h2>
<span class = "user">posted by: u/Lazypaul </span><span class = "numcomments">6 comments </span><a href="/r/bioinformatics/comments/lo84lt/learning_resources_on_gwas/go4odi9/" target="_blank">(source)</a>
<button onclick="toggles(681)">View full post</button><div id="681" class="mdwrapper"><div class="md"><p>I made a video series on youtube called &quot;Genome-wide association studies explained simply&quot;  </p>

<p><a href="https://www.youtube.com/watch?v=-IMZArULqbs&amp;list=PLwq1fhnu7450b7NoSc-Dgn1-PZFedtWq2&amp;ab_channel=BehaviouralGenomics">https://www.youtube.com/watch?v=-IMZArULqbs&amp;list=PLwq1fhnu7450b7NoSc-Dgn1-PZFedtWq2&amp;ab_channel=BehaviouralGenomics</a></p>
</div></div>
</div>
<div class="post">
<h2>How to start learning bioinformatics from absolute zero?</h2>
<span class = "user">posted by: u/didicoyrazorgang </span><span class = "numcomments">20 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/lnto19/how_to_start_learning_bioinformatics_from/" target="_blank">(source)</a>
<button onclick="toggles(682)">View full post</button><div id="682" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>I would like to learn bioinformatics, however, I don&#39;t have any prior knowledge in molecular biology or programming, and never had an experience in a wet-lab or a dry-lab. Where do I begin to learn bioinformatics?</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>How to start learning bioinformatics from absolute zero?</h2>
<span class = "user">posted by: u/CanonCopy </span><span class = "numcomments">20 comments </span><a href="/r/bioinformatics/comments/lnto19/how_to_start_learning_bioinformatics_from/go2b576/" target="_blank">(source)</a>
<button onclick="toggles(683)">View full post</button><div id="683" class="mdwrapper"><div class="md"><p>Try Galaxy, a free place to do bioinformatics via a webpage. Search for &quot;Galaxy Training Network&quot; to see some introductory tutorials. This will give concepts of both molec biology and bioinformatics. From there, if you want to, you can move into learning Unix on the command line - search for &quot;Software Carpentry&quot; for intro tutorials.</p>
</div></div>
</div>
<div class="post">
<h2>How to start learning bioinformatics from absolute zero?</h2>
<span class = "user">posted by: u/zemaxe </span><span class = "numcomments">20 comments </span><a href="/r/bioinformatics/comments/lnto19/how_to_start_learning_bioinformatics_from/go3qbz2/" target="_blank">(source)</a>
<button onclick="toggles(684)">View full post</button><div id="684" class="mdwrapper"><div class="md"><p>If you want a complete education, here it is, just stick to it:</p>

<p><a href="https://github.com/ossu/bioinformatics">https://github.com/ossu/bioinformatics</a></p>
</div></div>
</div>
<div class="post">
<h2>Master in Bioinformatics</h2>
<span class = "user">posted by: u/ezits </span><span class = "numcomments">20 comments </span><a href="/r/bioinformatics/comments/loay7w/master_in_bioinformatics/go4z4gf/" target="_blank">(source)</a>
<button onclick="toggles(685)">View full post</button><div id="685" class="mdwrapper"><div class="md"><p>I did my undergrad in bio and MS in bioinformatics and I loved it!! 
While my undergrad gave me all the conceptual background I use in my job, I learned all of the hard, technical skills in my masters (like bash scripting, computational genomic pipelining, database admin, etc).</p>

<p>My best piece of advice is to start taking those coding classes as early as you can, and don’t be afraid of higher level math and machine learning! I definitely shied away from those topics for longer than I should have, only to learn that they’re not as intimidating as I feared. </p>

<p>I also have my opinions about how intro genetics is taught in undergrad, and would suggest exploring more advanced realms (evolutionary genomics, statistical genetics, etc) before or during your masters.</p>
</div></div>
</div>
<div class="post">
<h2>Master in Bioinformatics</h2>
<span class = "user">posted by: u/None </span><span class = "numcomments">20 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/loay7w/master_in_bioinformatics/" target="_blank">(source)</a>
<button onclick="toggles(686)">View full post</button><div id="686" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>Hello,</p>

<p>I’m currently a 2nd year Biological Science student and I’m interested in genomics and bioinformatics and I  am considering pursuing a master degree in bioinformatics after graduation. Is there anyone who’s done a master in bioinformatics that could give like an overview, insight or experience undergoing their master.</p>

<p>Thank you.</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>What sort of experiences to look for as an undergrad planning to Master in Bioinformatics?</h2>
<span class = "user">posted by: u/spez_edits_thedonald </span><span class = "numcomments">9 comments </span><a href="/r/bioinformatics/comments/lolxvg/what_sort_of_experiences_to_look_for_as_an/go7mxt3/" target="_blank">(source)</a>
<button onclick="toggles(687)">View full post</button><div id="687" class="mdwrapper"><div class="md"><blockquote>
<p>I’ll be learning how to analyze a large genetic dataset using some bioinformatics programs.</p>
</blockquote>

<p>nice</p>

<blockquote>
<p>Proficient in Java and Python</p>
</blockquote>

<p>NICE</p>

<blockquote>
<p>I am wondering what sort of experiences I should seek out for the summer and any future summers?</p>
</blockquote>

<p>check out UCLA BIG (bruins in genomics) program if not on your radar, prob would app to a lot of stuff like that, maybe some big companies, and some cool startups</p>

<blockquote>
<p>Should I be looking at pure software experiences to balance out and gain more knowledge in that field before zeroing in on bioinformatics? Are industry internships more valuable than simply building my skill set off of research opportunities if I’m looking to go into industry one day?</p>
</blockquote>

<p>Current trajectory sounds good, no need to do a deep dive on software before starting out, can always do that later. Industry and academic positions would both be good here, you can&#39;t go wrong. Depends on your interests. Industry jobs won&#39;t look down on academic experience. Academia sometimes has some slightly negative sentiment toward industry (because they&#39;re the ones that didn&#39;t choose industry jklmao). </p>

<blockquote>
<p>I’m also scared to apply for industry internships because I feel like I don’t know enough to really make an impact at a company.</p>
</blockquote>

<p>nah this doesn&#39;t matter, def reach out to companies if interested in industry</p>
</div></div>
</div>
<div class="post">
<h2>What sort of experiences to look for as an undergrad planning to Master in Bioinformatics?</h2>
<span class = "user">posted by: u/basheerbecerra </span><span class = "numcomments">9 comments </span><a href="/r/bioinformatics/comments/lolxvg/what_sort_of_experiences_to_look_for_as_an/go6sltt/" target="_blank">(source)</a>
<button onclick="toggles(688)">View full post</button><div id="688" class="mdwrapper"><div class="md"><p>If you are proficient in Java and Python, then you should mostly be fine in regards to starting a new bioinformatics project. Some people come in from complete scratch. Probably getting familiar with command line and R wouldn’t hurt. And familiar with the common bioinformatics formats i.e vcf, bam, etc would be helpful.</p>

<p>In my opinion, doing bioinformatics in industry or academia should both be quite valuable. Even without prior bioinformatics experience, you’ll be surprised how quickly you can pick up the necessary skills. </p>

<p>Bioinformatics is such a vast field that it will be hard to feel prepared. Best way is to do as many projects along your research interests and you’ll surely acquire the skills you need over time. You’ll also get better at quickly learning new technologies over time as well.</p>

<p>I think another good exercise is finding bioinformatics papers along your interest. Go through the process of understanding the experimental design of their work, the methods they used (especially tools), and maybe the supplementary code if provided. This can get you more familiar with the field and give you some insight on what tools and technologies you should learn.</p>
</div></div>
</div>
<div class="post">
<h2>PhD Grad School Decision Advice</h2>
<span class = "user">posted by: u/AJs_Sandshrew </span><span class = "numcomments">31 comments </span><a href="/r/bioinformatics/comments/lq2j3d/phd_grad_school_decision_advice/goeswxg/" target="_blank">(source)</a>
<button onclick="toggles(689)">View full post</button><div id="689" class="mdwrapper"><div class="md"><p>Top priorities are, in my opinion:</p>

<ol>
<li><strong>Funding</strong> - Don&#39;t pay for your degree. Non-negotiable.<br/></li>
<li><strong>Location</strong> - This is my personal opinion, but I highly weigh a place where you will enjoy living. You will essentially be spending the next 4+ years of your life in this location, so it might as well be a place you like. Grad school is a grueling process, and being able to disconnect from your research and live a happy life in your local environment is often overlooked.<br/></li>
<li><strong>Program that matches your interests</strong> - emphasis on program here (see next)<br/></li>
<li><strong>Specific Professors</strong> - the reason for selecting a <em>program</em> that fits your interests is so that you don&#39;t put all your eggs in one basketcase....er....professor. There is no guarantee that the specific lab you want to work in will have funding for you. And even times when they claim they will have funding may not be true when you finally start your program. Then you are potentially forced to join a lab that you might not want to join. Find a department where there are multiple labs/professors you can see yourself working with.<br/></li>
<li><strong>Prestige of University</strong> - this more so depends on your long-term goals. If you are aiming to stay in academia and shoot for a TT position, then this might be higher up on the priority list. A higher prestige PhD will help for a professorship, but is not required.</li>
</ol>
</div></div>
</div>
<div class="post">
<h2>PhD Grad School Decision Advice</h2>
<span class = "user">posted by: u/oldblue2021 </span><span class = "numcomments">31 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/lq2j3d/phd_grad_school_decision_advice/" target="_blank">(source)</a>
<button onclick="toggles(690)">View full post</button><div id="690" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>Hello everyone. I have just received the majority of my graduate school decisions and have gotten good news from several great PhD programs. I thought I&#39;d ask the people on this subreddit for some advice/insight about these programs. In no particular order, the programs I am seriously considering attending are:</p>

<p>- Dartmouth Quantitative Biomedical Sciences (full funding + NIH fellowship)</p>

<p>- Duke Computational Biology &amp; Bioinformatics (full funding)</p>

<p>- UNC Chapel Hill Biological &amp; Biomedical Sciences (full funding)</p>

<p>- UC Santa Cruz Biomolecular Engineering &amp; Bioinformatics (full funding + QB3 fellowship)</p>

<p>- U of Washington Biomedical Informatics &amp; Medical Education (no funding)</p>

<p>- U of Michigan Bioinformatics (full funding)</p>

<p>- UC Irvine Mathematical, Computational, and Systems Biology (Haven&#39;t heard back yet, but thought I&#39;d include it)</p>

<p>- U of Wisconsin Biomedical Data Science (Waitlist)</p>

<p>My research interests are biomedical informatics (precision medicine, omics data-type integration, disease/treatment outcome prediction, etc.). I am planning on visiting most of these schools before I make my final decision, but regardless I would appreciate any insight/info from all the smart people on this page. This is a big decision and I&#39;m trying to get as much info as possible. Any and all comments would be very much appreciated. Thank you.</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>General skills to start developing when new to bioinformatics</h2>
<span class = "user">posted by: u/frausting </span><span class = "numcomments">22 comments </span><a href="/r/bioinformatics/comments/jyvhx8/general_skills_to_start_developing_when_new_to/gd88fen/" target="_blank">(source)</a>
<button onclick="toggles(759)">View full post</button><div id="759" class="mdwrapper"><div class="md"><p>Agree with all of this. For the first suggestion of learning the command line and shell scripting, I HIGHLY recommend Vince Buffalo’s book <em>Bioinformatics Data Skills</em>. I only wish I had found it sooner. Such a great resource for learning the computational skills for the day to day work of being a bioinformatician.</p>
</div></div>
</div>
<div class="post">
<h2>What is the most aesthetically pleasing figure you’ve seen from a scientific paper?</h2>
<span class = "user">posted by: u/o-rka </span><span class = "numcomments">17 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/jd3k0x/what_is_the_most_aesthetically_pleasing_figure/" target="_blank">(source)</a>
<button onclick="toggles(787)">View full post</button><div id="787" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>I’m wondering what fellow bioinformatics view as beautiful figures.  If there’s any that pop out at you, can you link it out with the publication?</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>Best IDE and packages to use for python in bioinformatics</h2>
<span class = "user">posted by: u/Riddy_93 </span><span class = "numcomments">18 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/iblw4x/best_ide_and_packages_to_use_for_python_in/" target="_blank">(source)</a>
<button onclick="toggles(841)">View full post</button><div id="841" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>I am new to the python programming language and am currently using Sublime test, can anyone recommend me any other better IDE for python and which packages would you recommend for bioinformatics.</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>Advice on choosing a supervisor/thesis lab(Masters Student)</h2>
<span class = "user">posted by: u/yeetymathynerd </span><span class = "numcomments">6 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/hzxkls/advice_on_choosing_a_supervisorthesis_labmasters/" target="_blank">(source)</a>
<button onclick="toggles(866)">View full post</button><div id="866" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>I am currently scoping labs for my masters thesis and I am running into a problem where most of the lab information I am finding seems a little outdated and I cant really assess the lab size and current research being done. I also can not access a lot of the faculty in person, something I would prefer, due to COVID and teleworking. Any recommendations on how I can get around this without cold emails since my program requires me to set up my own rotation supervisors? </p>

<p>&#x200B;</p>

<p>Also, what are some things I should be asking PIs about that yall think is important but not stressed or often forgotten? What are things yall regret not knowing before a rotation or getting into a certain lab?</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>What are some good bioinformatics PhD programs I should apply to?</h2>
<span class = "user">posted by: u/USSr90 </span><span class = "numcomments">15 comments </span><a href="/r/bioinformatics/comments/hv5yux/what_are_some_good_bioinformatics_phd_programs_i/fyrt4fg/" target="_blank">(source)</a>
<button onclick="toggles(883)">View full post</button><div id="883" class="mdwrapper"><div class="md"><p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4944079/">https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4944079/</a></p>
</div></div>
</div>
<div class="post">
<h2>Valuable R skills and packages</h2>
<span class = "user">posted by: u/deltawhiskey007 </span><span class = "numcomments">32 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/hnxp57/valuable_r_skills_and_packages/" target="_blank">(source)</a>
<button onclick="toggles(896)">View full post</button><div id="896" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>Hi everyone, I am currently a second year undergrad biomedical science student learning how to use R. I am hoping to use these skills to get lab positions and work experience in the field. Are there any particular things I should focus on or packages that I should get familiar with using in R that are valuable in bioinformatics/biochemistry field? </p>

<p>Im in North America if that is at all relevant to these questions.</p>

<p>Thanks</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>I have metatranscriptomes from the human microbiome and would like to associate microbes with gene expression and patient characteristics. What are the appropriate general approaches and statistical tests for asking this questions?</h2>
<span class = "user">posted by: u/Silenci </span><span class = "numcomments">1 comments </span><a href="https://www.reddit.com/r/bioinformatics/comments/hgk3zb/i_have_metatranscriptomes_from_the_human/" target="_blank">(source)</a>
<button onclick="toggles(912)">View full post</button><div id="912" class="mdwrapper"><!-- SC_OFF --><div class="md"><p>Hi all,</p>

<p>As I mentioned in the title, I have a metatranscriptome and want to ask several questions relating the microbiome with the characteristics of the patient/sample. All of these questions will start assuming I have the relative abundance of all bacteria in the metatranscriptome:</p>

<p>1) Test if <em>a specific</em> bacterium is associated with a discrete patient characteristic (e.g. country of origin).<br/>
2) Test if <em>a specific</em> bacterium is associated with a continuous patient characteristic (e.g. age).<br/>
3) Test if <em>a specific</em> bacterium is associated with the expression of any genes.</p>

<p>What are the general approaches and statistical tests I would use to address these questions? I suppose one way to do it would be to split the samples into &quot;bacteria-of-interest high&quot; and &quot;bacteria-of-interest low&quot; samples, and compare the characteristic/do differential gene expression between the groups, but not sure this is the most robust approach.</p>

<p>I&#39;d also be interested in similar comparisons, but rather than a specific bacteria associating <em>any bacteria</em> with a patient characteristic or gene expression. </p>

<p>Thank you very much!</p>
</div><!-- SC_ON --></div>
</div>
<div class="post">
<h2>Is there any online community for connectomics?</h2>
<span class = "user">posted by: u/Nemiara </span><span class = "numcomments">15 comments </span><a href="/r/bioinformatics/comments/hchohi/is_there_any_online_community_for_connectomics/fvf898g/" target="_blank">(source)</a>
<button onclick="toggles(919)">View full post</button><div id="919" class="mdwrapper"><div class="md"><p>A pretty small community. There&#39;s Neurostars for any specific questions regarding neuroinformatics. Honestly that&#39;s all Ive come across in the past half year of a connectomics related project. I&#39;m sure there&#39;s something more out there, but Neurostars is all I know.</p>
</div></div>
</div>
</div>
<script src="../assets/js/script.js"></script>
</body></html>